src/constant/index.js

```diff
@@ -291,3 +291,41 @@ export const CROSS_SELL_PRODUCT_IDS_MAP = IS_TESTNET
         'likenft1r8mce5h96nvedtm7rh9va5mda6fgysqfprlxftuef09s8kfjt2jsqeyyjq',
       ],
     };
+
+export const AFFILIATION_CHANNELS = [
```

Comment:
[🎨 Fetch channel displayName for @likerid](https://github.com/likecoin/liker-land/pull/1751/commits/e55755861482bcf3d881a6134940d840df092ff5)

Reply:
If we now know how to fetch channel info from Liker ID, `AFFILIATION_CHANNELS` here should just be a map of legacy string channel ID => Liker ID
Then we can fetch everything using a same logic (from Liker ID API), instead of hard coding names here
On the other hand if the channel already uses liker id as string, we don't need to hardcode it here

Reply:
Can @nwingt provide a list of known legacy string in use to hardcode here?

Reply:
Also we can safely not include liker_land in this list